server/routes/index.js

```diff
@@ -64,6 +64,7 @@ module.exports = function Index ({ authenticationMiddleware }) {
     }
     req.session.currentView = params.subsection
     req.session.caseListDate = params.date
+    req.session.backLink = `/${params.courtCode}/cases/${params.date}?page=${templateValues.params.page}`
```

Comment:
We should probably add a test for this, give me a shout tomorrow if you need help with it.